src/main/java/com/example/bookshop/service/impl/UserServiceImpl.java

```diff
@@ -36,4 +39,12 @@ public UserRegistrationResponseDto register(UserRegistrationRequestDto request)
         User savedUser = userRepository.save(user);
         return userMapper.toUserResponse(savedUser);
     }
+
+    public User getUserFromContext() {

```

Comment:
can be private

Reply:
I don't want to make it private because I want to have an opportunity to use it in different services.